src/test/java/agh/ics/oop/AnimalTest.java

```diff
@@ -0,0 +1,93 @@
+package agh.ics.oop;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class AnimalTest {
+
+    // in doesTurn tests, boolean assertion is probably sufficient,
+    // but there's also an equals assertion in each one, just in case
```

Comment:
na pewno fajnie prócz pozycji sprawdzać też orientację i w sumie dokładnie to ten equals robi (przy czym można by zamiast tego po prostu dodać getter dla orientacji)